### PR TITLE
Eric admin UI

### PR DESCRIPTION
**Pull request by lobsterdude3**
### Summary 

This pull request is the first step towards implementing the admin panel

- [x] install necessary libraries (react, material UI)
- [x] Added beginning of UI with the sidebar, and app bar.

Remaining TODOs:

- [ ] Add icons for sidebars
- [ ] Change styling

### Test Plan 

Launch code using npm start and play around with UI

### Notes 

https://gyazo.com/841ded7ad400156494048674bf6941db

screenshot of UI

### CODE REVIEW

File: admin/src/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Box
+      sx={{ flexGrow: 1, bgcolor: 'background.paper', display: 'flex', height: 224 }}
+    >
```
Comment (markd88) [inline admin/src/Sidebar.tsx:50]: It will be good if the code is more compact

Comment (markd88) [inline admin/src/Sidebar.tsx:50]: Leave some blanks to make the code more structured

**Review by markd88 — APPROVED** [review]
The code is good

File: admin/src/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <Tabs
+        orientation="vertical"
+        variant="scrollable"
+        value={value}
+        onChange={handleChange}
+        aria-label="Vertical tabs example"
```
Comment (neketka) [inline admin/src/Sidebar.tsx:56]: Make sure to remove the sample code from the tutorial and name it appropriately.

**Review by neketka — CHANGES_REQUESTED** [review]

**Review by neketka — APPROVED** [review]

**Pull request merged**